pkgs/by-name/ea/eaphammer/package.nix

```diff
@@ -0,0 +1,80 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  python3Packages,
+  makeWrapper,
+  apacheHttpd,
+  curl,
+  dnsmasq,
+  hcxdumptool,
+  hcxtools,
+  libnfnetlink,
+  libnl,
+  libpcap,
+  openssl,
+  responder,
```

Comment:
Responder might also be referenced somewhere in the source code, may need to be patched as well to be properly working.

I'm not sure if there is hostapd in `nixpkgs` right now and maybe it's needed to override to use the specific one compatible with `eaphammer`, I believe it is `hostapd` version 2.6

Reply:
It is mentioned that the specific `hostapd` to use is already contained in `eaphammer`'s repository : 

> [DEPRECATED UNTIL FURTHER NOTICE... use hostapd from s0lst1c3/eaphammer repo] Hostapd 2.6 patched with a trimmed version of hostapd-wpe for use in eaphammer